javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+const form = document.getElementById('contact-form');
+const formValues = {};
+
+form.addEventListener('submit', function (e) {
+  const formElements = document.getElementById('contact-form').elements;
```

Comment:
No need for another query selector here, you already queried for `form` on line 1. You can just do `form.elements`